rust/tensors.rs

```diff
@@ -108,6 +115,7 @@ impl<'a> TensorDescriptor<'a> {
         // accidentally outlive the original tensor.
         unsafe {
             TensorDescriptor {
+                name: CStr::from_ptr(tensor.name),
```

Comment:
We can assume this is never null, can't we?

Reply:
Good question.. I don't think we can assume that..  but we can change librunecoral to return an empty string if it is null?